playbooks/phobos_vpn.yml

```diff
@@ -33,3 +33,27 @@
         service: vpnc.service
         daemon_reload: yes
         state: restarted
+      failed_when: false
```

Comment:
Instead of using failed_when: false, rather use a try/rescue/always block. You could make the diagnostic output only show in the rescue block, so it'll only show when it fails the try block which is where the service restart should be.

Reply:
yeah, i stumbled upon that yesterday (while reminding myself the syntax of how to put a conditional on a block): https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html#error-handling.  i just didn't have the time at that moment to learning something new and experiment with it (not knowing how long the then-current vpnc failures was going to last).

all that is not to say that's it's not the better approach, and i'll implement it when i have the time (and the opportunity to test it with vpnc failing), but merely to justify the less-then-optimal change in this PR.